.github/actions/install_node_dependencies/action.yml

```diff
@@ -0,0 +1,26 @@
+name: install_node_dependencies
```

Comment:
Don't need the actions in this folder anymore, right?

Reply:
So you are suggesting to move all the actions under `.github/workflows/` or create a actions folder under `.github/workflows/` like this `.github/workflows/actions` ?

Reply:
I think `setup_python_env` is not required because `actions/setup-python@v5` already sets up python, and the other two actions can just be steps in the workflow.

Reply:
Oh wait, it installs test requirements, you can rename the action instead.

Reply:
Actually I need to rename it correctly instead of setup python it should be "Install Python dependencies"

```
name: setup_python_env
description: Setup Python environment
runs:
  using: composite
  steps:
  - name: Install Python dependencies
    run: |-
      pip install git+https://github.com/kedro-org/kedro@main
      pip install -r package/test_requirements.txt -r demo-project/src/docker_requirements.txt -U
    shell: bash
  - name: Echo package versions
    run: |-
      python -V
      pip freeze
    shell: bash
```

Reply:
It's upto you actually, I think having them as reusable actions is also not a bad setup!

Reply:
> It's upto you actually, I think having them as reusable actions is also not a bad setup!

It is about "Don't need the actions in this folder anymore, right?" ?

Reply:
We just have these as steps in kedro but maybe it makes more sense to have them as actions and we can do smth like this in Kedro too. ^